### PR TITLE
refactor: review from fury

### PR DESCRIPTION
**Pull request by black-wyvern-dev**
Looks the repos is well-structured & cleanly coded.
Left review in this PR and also here are detail documents.
https://docs.google.com/document/d/1vbbAzG4FsID7T1fHRki2LdtoJGjtmJExrnGlFOf_JLU/edit

### CODE REVIEW

File: src/helpers/nft.ts
Comment (black-wyvern-dev) [inline src/helpers/nft.ts:1]: Should isolate Components & Funtionalities interacting with API/Web3.

File: src/hooks/useModal.tsx
Comment (black-wyvern-dev) [inline src/hooks/useModal.tsx:1]: Consistency is one of important thing which every FE devs have to do.
I can see type defined Nullable, but here used origin null types too.

File: src/index.tsx
Comment (black-wyvern-dev) [inline src/index.tsx:1]: Add single empty lines to make looks clean.

File: src/logo.svg
Comment (black-wyvern-dev) [inline src/logo.svg:1]: Can't find use this svg file anywhere.
And the asset images should be located in `public` dir or should make isolate `src/assets/img` dir.

File: src/pages/Home/Home.tsx
Comment (black-wyvern-dev) [inline src/pages/Home/Home.tsx:1]: Same here.
Use callback hook & constants from single definition place.

File: src/pages/Nft/Nft.tsx
Comment (black-wyvern-dev) [inline src/pages/Nft/Nft.tsx:1]: Immigrated the functionalities interacting with API/Web3 as isolate utils.
UI Component should keep in UI side facts like render, state, effect ...

**Review by black-wyvern-dev — COMMENTED** [review]
The demo looks well-structured and has no error anywhere.
TailwindCSS is highly recommended library, and looks responsive design is implemented too.
If you understand and design everything in this project, I think it's enough to continue making money further.

File: package.json [hunk 1]
```diff
@@ -3,13 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.16.5",
```
[the remark this answers is not in the document]
Comment (sven3270350) [reply package.json:6]: prod & dev is good for this. Thanks.

File: public/robots.txt
[the remark this answers is not in the document]
Comment (sven3270350) [reply public/robots.txt:1]: Yeah, That is simply copied one. I will delete it.

**Pull request merged**